packages/core/src/components/Dropdown/Dropdown.example.jsx

```diff
@@ -0,0 +1,48 @@
+import Dropdown from './Dropdown';
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+const dropdownOptions = [
+  { label: 'A', value: 'A' },
```

Comment:
I'd like to change these to `Option 1`, `Option 2`to keep the examples consistent